nuleaf-source/controllers/events.js

```diff
@@ -29,11 +31,14 @@ exports.search = function(req, res) {
     start_date: req.query.start_date,
     end_date  : req.query.end_date,
     location  : req.query.location,
+    sort      : req.query.sort,
+    sortBy    : req.query.sortBy,
     skip      : req.query.skip,
     limit     : req.query.limit
   }, function(err, events) {
     if (err) { return res.status(500).json({ error: err }); }
-    return res.status(200).json(events);
+    return res.status(200
```

Comment:
I think you fixed this in your last PR, not sure what happened but can you inline this again?